src/confcom/setup.py

```diff
@@ -18,7 +18,7 @@
 
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
 
-VERSION = "0.3.6"
+VERSION = "1.0.0"
```

Comment:
`1.0.0` is the GA version, if you still need beta version, you can use `1.0.0b1`

Reply:
GA is good for us, thanks for checking 😊